pallets/phala/src/stakepool.rs

```diff
@@ -613,6 +666,38 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(0)]
+		pub fn check_and_maybe_force_withdraw(
+			origin: OriginFor<T>,
+			pid: u64,
+		)  -> DispatchResult {
+			ensure_signed(origin)?;
+			let now = <T as registry::Config>::UnixTime::now()
+				.as_secs()
+				.saturated_into::<u64>();
+			let mut pool = Self::ensure_pool(pid).expect("Stake pool must exist; qed.");
+			Self::try_process_withdraw_queue(&mut pool);
+			let grace_period = T::GracePeriod::get();
+			let mut releasing_stake = Zero::zero();
+			for worker in pool.cd_workers.iter() {
+				let miner: T::AccountId = pool_sub_account(pid, &worker);
+				let stakes: BalanceOf<T> = mining::pallet::Stakes::<T>::get(&miner).ok_or(mining::Error::<T>::CDWorkerNoStake)?;
```

Comment:
Please add a TODO here. This is currently good, but once we introduce slash, the actual releasing stake could be less than that in `Stakes`.

Something like: `// TODO(mingxuan): handle slash`